.travis/run.sh

```diff
@@ -15,9 +15,4 @@ fi
 
 source ~/.venv/bin/activate
 
-if [[ "$(uname -s)" == "Darwin" ]]; then
-    # no fakeroot on OS X
-    sudo tox -e $TOXENV -r
-else
-    fakeroot -f scripts/faked-debug.sh -u tox -r
-fi
+test-wrapper/target/release/test-wrapper tox -r -e "$TOXENV"
```

Comment:
I'm not sure, I think it'll disable some tests if we don't use sudo. Should we?

I also just realized that's the reason `-e "$TOXENV"` was needed for OSX. I'll remove that.